tools/webpack/shared.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+const TerserPlugin = require( 'terser-webpack-plugin' );
+const postcss = require( 'postcss' );
+const { join } = require( 'path' );
+
+const baseDir = join( __dirname, '../../' );
+
+const baseConfig = ( env ) => {
+	const mode = env.environment;
+
+	const config = {
+		target: 'browserslist',
+		mode,
+		optimization: {
+			moduleIds: mode === 'production' ? 'deterministic' : 'named',
+			minimizer: [
+				new TerserPlugin( {
+					extractComments: false,
```

Comment:
To keep the License comments inside the minimized files we need to set this.

Omitting this will create a `.min.LICENCE.txt` file that contains the licenses. Seehttps://webpack.js.org/plugins/terser-webpack-plugin/#extractcomments

Would be great to enable this to make the minimized files smaller, but since they are inlined today I opted to still have them inline.

Reply:
Related PRs: https://github.com/WordPress/gutenberg/pull/22990 and https://github.com/WordPress/gutenberg/pull/28231. @swissspidy, do we need the same handling in WordPress core? We have non-minified files included, too.

Reply:
`extractComments: false` makes sense.

Using the same config as Gutenberg to keep translator comments and avoid mangling i18n functions also makes sense